include/boost/graph/adjacency_list_io.hpp

```diff
@@ -76,7 +76,7 @@ void get
 }
 
 // get a subset of properties tagged Stag
-template<class Tag, class Value, class Next, 
+template<class Tag, class Value, class Next,
```

Comment:
Does this file contain anything other than whitespace changes?   It's *very* hard to review stuff like this.... sorry!